onebusaway-enterprise-webapp/src/main/java/org/onebusaway/enterprise/webapp/actions/api/SearchResultFactoryImpl.java

```diff
@@ -108,7 +114,21 @@ public SearchResult getRouteResult(RouteBean routeBean) {
         if(routeHasVehiclesInService) {
       	  hasUpcomingScheduledService = true;
         }
-
+        
+        try {
+        	if(routeBean != null) {
+				    String new_trip = _tConfigurationServiceClientFileImpl.getItem("trip", routeBean.getId(), 
```

Comment:
Please take a look at current coding conventions.   OBA uses camelCasing in variables.